consensus/src/messages/handlers.rs

```diff
@@ -206,7 +206,7 @@ impl RequestMissingBlocks {
         blockchain: &BlockchainProxy,
     ) -> Result<ResponseBlocks, ResponseBlocksError> {
         // TODO We might want to do a sanity check on the locator hashes and reject the request if
-        //  they they don't match up with the given target hash.
+        // they they don't match up with the given target hash.
```

Comment:
This leading space indicates that the line belongs to the TODO above, making JetBrains IDEs highlight the whole comment as a TODO.